docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -69,19 +69,23 @@ For information about how to set this compiler option programmatically, see <xre
 |C# 1.2|[Download DOC](https://www.ecma-international.org/publications/files/ECMA-ST-ARCH/ECMA-334%202nd%20edition%20December%202002.pdf)|C# Language Specification Version 1.2: Microsoft Corporation|
 |C# 1.0|[Download DOC](https://www.ecma-international.org/publications/files/ECMA-ST-ARCH/ECMA-334%201st%20edition%20December%202001.pdf)|C# Language Specification Version 1.0: Microsoft Corporation|
 
-### Minimum compiler version needed to support all language features
-
-CS80: Microsoft Visual Studio/Build Tools 2019, version 16, or .NET Core 3.0 SDK  
-CS73: Microsoft Visual Studio/Build Tools 2017, version 15.7  
-CS72: Microsoft Visual Studio/Build Tools 2017, version 15.5  
-CS71: Microsoft Visual Studio/Build Tools 2017, version 15.3  
-CS7: Microsoft Visual Studio/Build Tools 2017  
-CS6: Microsoft Visual Studio/Build Tools 2015  
-CS5: Microsoft Visual Studio/Build Tools 2012 or bundled .NET Framework 4.5 compiler  
-CS4: Microsoft Visual Studio/Build Tools 2010 or bundled .NET Framework 4.0 compiler  
-CS3: Microsoft Visual Studio/Build Tools 2008 or bundled .NET Framework 3.5 compiler  
-ISO2: Microsoft Visual Studio/Build Tools 2005 or bundled .NET Framework 2.0 compiler  
-ISO1: Microsoft Visual Studio/Build Tools .NET 2002 or bundled .N Framework 1.0 compiler  
+## Minimum SDK version needed to support all language features
+
+The following table lists the the minimum version of the SDK with the C# compiler that supports the corresponding language version:
+
+|C# version|Minimum SDK version|
+|----------|-------------------|
+|C# 8.0| Microsoft Visual Studio/Build Tools 2019, version 16.3, or .NET Core 3.0 SDK |
```

Comment:
@BillWagner here is the only content change: C# 8.0 is officially supported starting with **16.3** (not 16, as currently in the doc) version of Visual Studio 2019, isn't it?

Reply:
Correct, and thanks for the note calling out the substantive change. That makes for easier reviewing.